File: app/src/main/java/cu/axel/smartdock/widgets/HoverInterceptorLayout.java
package cu.axel.smartdock.widgets;
import android.widget.LinearLayout;
import android.content.Context;
import android.util.AttributeSet;
import android.view.MotionEvent;

public class HoverInterceptorLayout extends LinearLayout
{
	public HoverInterceptorLayout(Context context, AttributeSet attrs)
	{
        super(context, attrs);
    }

    @Override
    public boolean onInterceptHoverEvent(MotionEvent event)
	{
		return true;
    }
}

File: app/src/main/java/cu/axel/smartdock/widgets/HoverInterceptorLayout.kt
package cu.axel.smartdock.widgets

import android.content.Context
import android.util.AttributeSet
import android.view.MotionEvent
import android.widget.LinearLayout

class HoverInterceptorLayout(context: Context, attrs: AttributeSet?) : LinearLayout(context, attrs) {
    override fun onInterceptHoverEvent(event: MotionEvent): Boolean {
        return true
    }
}
